aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -209,6 +209,266 @@ SignalR hub methods now support injecting services through dependency injection
 
 Hub constructors can accept services from DI as parameters, which can be stored in properties on the class for use in a hub method. For more information, see [Inject services into a hub](xref:signalr/hubs?view=aspnetcore-7.0&preserve-view=true#inject-services-into-a-hub)
 
+## Blazor
+
+### Handle location changing events
+
+In .NET 7, Blazor supports location changing events. This allows you to warn users about unsaved work or to perform related actions when the user performs a page navigation.
+
+For more information, see the following sections of the *Routing and navigation* article:
+
+* [Navigation options](xref:blazor/fundamentals/routing?view=aspnetcore-7.0#navigation-options)
+* [Handle/prevent location changes](xref:blazor/fundamentals/routing?view=aspnetcore-7.0#handleprevent-location-changes)
+
+### Empty Blazor project templates
+
+Blazor has two new project templates for starting from a blank slate. The new **Blazor Server App Empty** and **Blazor WebAssembly App Empty** project templates are just like their non-empty counterparts but without demonstration code. These empty templates only include a basic home page, and we've removed Bootstrap so that you can start with a different CSS framework.
+
+For more information, see the following articles:
+
+* <xref:blazor/tooling?view=aspnetcore-7.0>
+* <xref:blazor/project-structure?view=aspnetcore-7.0>
+
+### Blazor custom elements
+
+The [`Microsoft.AspNetCore.Components.CustomElements`](https://www.nuget.org/packages/microsoft.aspnetcore.components.customelements) package enables building [standards based custom elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements) using Blazor.
+
+For more information, see <xref:blazor/components/index?view=aspnetcore-7.0#blazor-custom-elements>.
+
+<!--
+
+    IN PROGRESS
+
+### Improve prerendering and authentication experience in Blazor
```

Comment:
This will likely be an out of band package or sample that will work with .NET 6 & .NET 7. You can remove it from this list as it really isn't a new .NET 7 feature.